client/blocks/login/index.jsx

```diff
@@ -141,6 +142,17 @@ class Login extends Component {
 		if ( ! prevProps.accountType && isPasswordlessAccount( this.props.accountType ) ) {
 			this.props.sendEmailLogin();
 		}
+
+		if (
+			this.props.isJetpackWooDnaFlow &&
+			this.props.requestError?.code === 'unknown_user' &&
+			this.props.usernameOrEmail.includes( '@' )
```

Comment:
If we’re being pedantic it’s possible to have MX records on the tld to get emails at user@tld (as opposed to user@domain.tld)

https://serverfault.com/questions/154991/why-do-some-tld-have-an-mx-record-on-the-zone-root-e-g-ai

In practice it’s probably not commonly done because it’d probably fail email validation everywhere 😬

Reply:
Good catch! I agree it's confusing. I changed it to use `emailValidator` to validate in f943980fa19859f5aae65309eaff27a4fdddc366.